lib/livebook/notebook/explore.ex

```diff
@@ -1,5 +1,23 @@
-defmodule Livebook.Notebook.Welcome do
-  livemd = ~s'''
+defmodule Livebook.Notebook.Explore.Utils do
+  @moduledoc false
+
+  @doc """
+  Defines a module attribute `attr` with notebook parsed from `markdown`.
+  """
+  defmacro defnotebook(attr, markdown) do
+    quote bind_quoted: [attr: attr, markdown: markdown] do
+      {notebook, []} = Livebook.LiveMarkdown.Import.notebook_from_markdown(markdown)
+      Module.put_attribute(__MODULE__, attr, notebook)
+    end
+  end
+end
+
+defmodule Livebook.Notebook.Explore do
+  @moduledoc false
+
+  import Livebook.Notebook.Explore.Utils
+
+  defnotebook(:introduction_to_livebook, ~s'''
```

Comment:
I think we should define those on disk. Something like

    defnotebook "Introduction to Livebook", "My description"

That would lookup for "Introduction to Livebook.livemd" and add the description. The file would be read into disk and a `@external_resource` annotation to it would be added. Thoughts?

Reply:
Sounds good to me! The title is already in the notebook, so you meant `defnotebook(file_name, description)` right?

Reply:
We would need to parse the title out though... because we need it to generate the cards. I guess that's ok though? 🤔

Reply:
Not really, we parse the livemd into `%Notebook{}` data structure, so the title is available to us, that's what we already do :D

Reply:
Also: I think `defnotebook` should just parse the notebook and define an attribute with `%Notebook{}`, while the description and other stuff can go directly in `notebook_infos()`.